asynciterator.ts

```diff
@@ -1304,6 +1305,36 @@ export class SimpleTransformIterator<S, D = S> extends TransformIterator<S, D> {
 export class MultiTransformIterator<S, D = S> extends TransformIterator<S, D> {
   private _transformerQueue: { item: S | null, transformer: Source<D> }[] = [];
 
+  /**
+   Creates a new `MultiTransformIterator`.
+   @param {module:asynciterator.AsyncIterator|Readable} [source] The source this iterator generates items from
+   @param {object|Function} [options] Settings of the iterator, or the transformation function
+   @param {integer} [options.maxbufferSize=4] The maximum number of items to keep in the buffer
+   @param {boolean} [options.autoStart=true] Whether buffering starts directly after construction
+   @param {module:asynciterator.AsyncIterator} [options.source] The source this iterator generates items from
+   @param {integer} [options.offset] The number of items to skip
+   @param {integer} [options.limit] The maximum number of items
+   @param {Function} [options.filter] A function to synchronously filter items from the source
+   @param {Function} [options.map] A function to synchronously transform items from the source
+   @param {Function} [options.transform] A function to asynchronously transform items from the source
+   @param {boolean} [options.optional=false] If transforming is optional, the original item is pushed when its mapping yields `null` or its transformation yields no items
+   @param {Function} [options.multiTransform] A function to asynchronously transform items to iterators from the source
+   @param {Array|module:asynciterator.AsyncIterator} [options.prepend] Items to insert before the source items
+   @param {Array|module:asynciterator.AsyncIterator} [options.append]  Items to insert after the source items
+   */
+  constructor(source: AsyncIterator<S>,
+              options?: MultiTransformOptions<S, D> |
+                        MultiTransformOptions<S, D> & ((item: S) => AsyncIterator<D>)) {
+    super(source, options as TransformIteratorOptions<S>);
+
+    // Set transformation steps from the options
+    if (options) {
```

Comment:
This is just new functionality? Or something that broke? (Going to keep it in any case.)

Reply:
Yep, that's just new. I found it annoying that I had to create dedicated classes just for doing small multi-transformations.